docs/scheme-api/geda-scheme.texi

```diff
@@ -2055,6 +2055,8 @@ The @var{level} should be one of the symbols @code{error},
 @code{critical}, @code{warning}, @code{message}, @code{info} or
 @code{debug}.
 
+Note: if @var{level} is set to @code{'error}, main program will terminate.
```

Comment:
According to the style of previous sentences, the apostrophe symbol (') is unnecessary here :-)